vignettes/using_pdf_and_word_template.Rmd

```diff
@@ -64,13 +102,13 @@ There are some new functions and coding conventions that help with knitting the
 * Note that Word output cannot support many kableExtra features, such as `kableExtra::kable_styling()`.
    + Set `warning=kable_warnings` chunk option, otherwise Word output will have unwanted warnings.
 
-## Figures
+### Figures
 
 * Only output one figure per code chunk.
 * Figure name will be chunk name plus "fig:".
     + i.e. if chunk name is "-stats" then the table label will be "fig:descriptive-stats".
```

Comment:
`i.e. if chunk name is “-stats” then the table label will be “fig:descriptive-stats”.`
"descriptive-stats"